x/programs/rust/sdk_macros/Cargo.toml

```diff
@@ -10,3 +10,4 @@ proc-macro = true
 proc-macro2 = "1.0.67"
 quote = "1.0.33"
 syn = { version = "2.0.37", features = ["full"] }
+unzip-n = "0.1.2"
```

Comment:
Are you doing this by hand instead of using `cargo add`?

Reply:
yea, should prob use `cargo add` tho